suites/portal/studyRegistrationTest.js

```diff
@@ -0,0 +1,107 @@
+Feature('Study Registration @requires-portal @requires-requestor @aggMDS @discoveryPage');
```

Comment:
I think this test should be only run for HEAL manifests, or more specifically, for any manifest that has `"studyRegistration": true,` in feature flags. Otherwise this test will be failing